tools/packaging/common/envoy_bootstrap.json

```diff
@@ -736,12 +736,69 @@
     {{ end }}
   ]
   {{ end }}
-  {{ if .outlier_log_path }}
+  {{ if or .outlier_log_path .lrs_server_addr }}
   ,
   "cluster_manager": {
+    {{- if .outlier_log_path}}
     "outlier_detection": {
       "event_log_path": "{{ .outlier_log_path }}"
     }
+    {{- end}}
+    {{- if and .outlier_log_path .lrs_server_addr }}
+    ,
+    {{- end}}
+    {{- if .lrs_server_addr }}
+    "load_stats_config": {
+      "api_type": "GRPC",
+      "transport_api_version": "V3",
+      "grpc_services": [
+        {
+          "google_grpc": {
+            "target_uri": "{{ .lrs_server_addr }}",
+            "stat_prefix": "googlegrpcxds",
```

Comment:
This is very specific to google - I think it's great to add LRS reporting, but it would be ideal to keep the google-specific configs separated or at least have a way to configure a non-google LRS backend ( i.e. using workload identity/mTLS, etc).  

There are few ways - maybe have the entire 'loads_stats_config' JSON blob sent as a config option ( which google specific code may set ) ? 

We do have a separate google-specific bootstrap file as well - and there are other vendor-specific integrations in the code, I think it's very reasonable to integrate with all major platforms - but needs to be a bit more decoupled.

Reply:
Make sense to me.
I have changed it to have the entire `load_stats_config` JSON blob.
Please take a look.